turbopack/crates/turbopack-browser/src/chunking_context.rs

```diff
@@ -389,10 +389,10 @@ impl ChunkingContext for BrowserChunkingContext {
     }
 
     #[turbo_tasks::function]
-    async fn chunk_group(
+    async fn chunk_group_multiple(
         self: Vc<Self>,
         ident: Vc<AssetIdent>,
-        module: ResolvedVc<Box<dyn ChunkableModule>>,
+        modules: Vec<ResolvedVc<Box<dyn ChunkableModule>>>,
```

Comment:
It would be better for incremental builds when this uses `Vc<ChunkableModules>` instead.

Reply:
👍 
I've had to add `Vc::cell` at each callsite of `chunk_group_multiple`, e.g.:
```rust
                        ssr_chunking_context.chunk_group_multiple(
                            base_ident.with_modifier(ssr_modules_modifier()),
                            Vc::cell(ssr_modules),
                            Value::new(current_ssr_availability_info),
                        )
```
I hope that doesn't screw with caching?